src/contexts/UserContext.jsx

```diff
@@ -19,10 +19,10 @@ function UserProvider ({ children }) {
         if (sessionUser && !user) {
             setUser(sessionUser);
         }
-    });
+    }, [user]);
 
     useEffect(() => {
-        window.localStorage.setItem('user', user);
+        window.localStorage.setItem('user', JSON.stringify(user));
```

Comment:
Make sure it won't break anything else :) 

Also, I saw someone commenting that `localStorage` is not so secure, so you'd probably check that with other devs.

Oh, its right above in line 17 😄

Reply:
lol yup I'm the one that put that. It's just a temp setup